### ISSUE TITLE
Autosubmit verification code when 6 digits are entered.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-two-factor#79]
Right now we require users to click "[Enable]" or hit the `enter` key to submit their verification code. I don't see a reason why we wouldn't automatically submit it when they have entered `6` digits. It would save them time and since the data is temporary they need not remember it.

## Screenshot
<img width="368" alt="Screen Shot 2023-03-10 at 2 22 38 PM" src="https://user-images.githubusercontent.com/1657336/224230616-6e862bfb-be50-49af-9805-33a6ff1bc79b.png">

**Comment by iandunn** [comment]
The upstream plugin recently made that change on the login screen. I do have [a UX concern](https://github.com/WordPress/two-factor/pull/518#discussion_r1110101003), but don't feel strongly. We're already using an XHR in this UI, so it's less of an issue anyway.

**Comment by StevenDufresne** [comment]
> a UX concern

Agreed. It should fail silently to not disrupt the user flow experience. 

- Check at 6 digits
  - If it works, provide success confirmation. 
  - If it fails, show errors when the user clicks the button, without doing another xhr call.
